src/app/labeling/components/labeling.component.ts

```diff
@@ -1389,7 +1393,31 @@ export class LabelingComponent implements OnInit, OnDestroy {
     }
   }
 
+  addLabelToMarkedTextFromSuggestion(overlayData) {
```

Comment:
no checks are made for the given weak supervision label. This means that the user could accept the same label multiple times or create an overlapping issue with other manual labels (or itself). 
![image](https://user-images.githubusercontent.com/91723236/190973637-0890bf0f-5d49-4a59-9e0c-555dc2854864.png)